packages/sdk/test/zora-fork-mint.test.ts

```diff
@@ -0,0 +1,199 @@
+import { selectors } from '@boostxyz/signatures/events';
+import {
+  loadFixture,
+  mine,
+  reset,
+} from '@nomicfoundation/hardhat-toolbox-viem/network-helpers';
+import {
+  http,
+  type Address,
+  type Hex,
+  createTestClient,
+  pad,
+  parseEther,
+  publicActions,
+  walletActions,
+} from 'viem';
+import { base } from 'viem/chains';
+import { beforeAll, beforeEach, describe, expect, test, vi } from 'vitest';
+import { BoostCore } from '../src/BoostCore';
+import {
+  type ActionStep,
+  FilterType,
+  PrimitiveType,
+  SignatureType,
+  StrategyType,
+  prepareSignerValidatorClaimDataPayload,
+} from '../src/utils';
+import { accounts } from './accounts';
+import {
+  type BudgetFixtures,
+  type Fixtures,
+  defaultOptions,
+  deployFixtures,
+  fundBudget,
+} from './helpers';
+
+let fixtures: Fixtures, budgets: BudgetFixtures;
+
+describe.skipIf(!process.env.VITE_ALCHEMY_API_KEY)(
+  'Boost with NFT Minting Incentive',
+  () => {
+    // This is the zora contract we're going to push a transaction against
```

Comment:
could simplify this function body by moving these constants outside the describe block. might be more readable